ts/src/header-validator/validate-json.ts

```diff
@@ -1213,6 +1213,40 @@ function aggregatableSourceRegistrationTime(
   return enumerated(ctx, j, AggregatableSourceRegistrationTime)
 }
 
+function warnInconsistentAggregatableKeys(ctx: Context, t: Trigger): void {
+  const triggerDataKeys = new Set<string>()
+
+  ctx.scope('aggregatable_trigger_data', () => {
+    for (const [index, datum] of t.aggregatableTriggerData.entries()) {
+      ctx.scope(index, () => {
+        for (const key of datum.sourceKeys) {
+          triggerDataKeys.add(key)
+
+          if (!t.aggregatableValues.has(key)) {
+            ctx.scope('source_keys', () =>
```

Comment:
If multiple keys in the same `source_keys` have warning message, probably can be consolidated to the same one? But it's fine leaving as is for readability.

Reply:
Ack.